vulnfeeds/utility/logginghelpers.go

```diff
@@ -0,0 +1,60 @@
+package utility
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"runtime/debug"
+
+	"cloud.google.com/go/logging"
+)
+
+// LoggerWrapper wraps the Logger provided by google cloud
+// Will default to the go stdout and stderr logging if GCP logger is not set
+type LoggerWrapper struct {
+	Logger *logging.Logger
+}
+
+// InfoLogf prints Info level log
+func (wrapper LoggerWrapper) InfoLogf(format string, a ...any) {
```

Comment:
nit: just Infof, Warnf, Fatalf etc.

The "Log" part is redundant because you're calling this from a logger instance that's going to be named "logger" or something similar anyway.